packages/manager/modules/web-universe-components/src/order/order.controller.js

```diff
@@ -0,0 +1,267 @@
+import get from 'lodash/get';
+import isEqual from 'lodash/isEqual';
+import isEmpty from 'lodash/isEmpty';
+import isString from 'lodash/isString';
+import some from 'lodash/some';
+import sortBy from 'lodash/sortBy';
+
+import {
+  CATALOG_ITEM_TYPE_NAMES,
+  ISO_DURATION_FORMAT,
+  ORDER_TYPES,
+} from './order.constants';
+
+export default class OrderController {
+  /* @ngInject */
+  constructor($translate, WucOrderCartService) {
+    this.$translate = $translate;
+    this.WucOrderCartService = WucOrderCartService;
+
+    this.currentIndex = 0;
+  }
+
+  $onInit() {
+    if (!this.catalog) {
+      throw new Error('WucOrderComponent requires a catalog');
+    }
+
+    this.confirmation = {};
+
+    this.bindings = {
+      catalog: this.catalog,
+      catalogItemTypeName: this.catalogItemTypeName,
+      productName: this.productName,
+      sendCurrentState: this.sendCurrentState,
+      serviceNameToAddProduct: this.serviceNameToAddProduct,
+      type: this.type,
+      user: this.user,
+    };
+
+    this.eventHandlers = {
```

Comment:
I understand it is easier to read if those are encapsulated but is it really needed ? It looks to me unecessary to copy all bindings especially if all eventHandlers are prefixed by `on`  it is redundant

Reply:
Apparently not, you just asked us to no start the name of an event handler by `on` 😄

Reply:
All `onMethods` are merged into `bindings`